### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Rick-Anderson**
Kill dead wood

skipciplease
skip ci please

### CODE REVIEW

**Comment by livarcocc** [comment]
@richlander can you take a look as well?

**Comment by johnbeisner** [comment]
@mmitche 
Why is this PR running the checks when we have a "skipciplease" designate in the PR?

**Comment by mmitche** [comment]
@johnbeisner It did skip CI.  The checks "waiting for status" are added by GitHub itself.  You simply told jenkins not to run.

**Comment by mmitche** [comment]
@dotnet/dnceng FR FYI if you ever see this.

File: README.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ Looking for V1 of the .NET Core tooling?
 
 If you are looking for the v1.0.1 release of the .NET Core tools (CLI, MSBuild and the new csproj), head over to https://dot.net/core and download!
 
-> **Note:** the release/2.0.0 branch of the CLI repo is based on the upcoming v2 of .NET Core and is considered pre-release. For production-level usage, please use the
+> **Note:** the release/2.0.0 branch of the CLI repo is based on the upcoming v2 of .NET Core and is considered pre-release. For production-level usage, use the
```
Comment (MattGal) [inline README.md]: It's been a long time since my tech writing course, I'd love to understand why please needs to be deleted here

Comment (Rick-Anderson) [inline README.md]: Other than dead-wood? For something like this (production) - you don't want to imply it's options but will make me happy.
 cc @tdykstra
It doesn't **need** to be removed. Removing improves the writing.

Comment (tdykstra) [inline README.md]: Per the style guide:

> Avoid *please* except in situations where the customer is asked to do something inconvenient or the application or site is to blame for the situation.

**Pull request merged**